util/saconfig

```diff
@@ -96,39 +104,45 @@ def set_zero(dev, channels):
     if not channels:
         return
 
-    logging.info("Setting scale offsets. Current position: %s", dev.position.value)
+    # logging.info("Setting scale offsets. Current position: %s", dev.position.value)
+    # for channel in channels:
+    #     logging.info("Setting the scale offset for channel %d", channel)
+    #     scale_offset = dev.GetProperty_i64(smaract.SA_CTLDLL.SA_CTL_PKEY_LOGICAL_SCALE_OFFSET, channel)
+    #     position = dev.GetProperty_i64(smaract.SA_CTLDLL.SA_CTL_PKEY_POSITION, channel)
+    #     # set the new scale offset
+    #     dev.SetProperty_i64(smaract.SA_CTLDLL.SA_CTL_PKEY_LOGICAL_SCALE_OFFSET, channel, scale_offset + position)
 
+    logging.info("Setting current position to 0. This automatically adjusts the logical scale offset.")
```

Comment:
Could you still log the current position? It can be handy to know by how much it shifted.